.gitignore

```diff
@@ -1,3 +1,5 @@
+/vendor/
 /nginx/servers/*
 !/nginx/servers/proximis.conf.dist
 /.env
+composer.lock
```

Comment:
Why do you need to ignore the `composer.lock`?

Reply:
Because i use symfony/filesystem for `docker-upgrade` file